### DISCUSSION TITLE
Missing XML fragment xl/sharedStrings.xml

### DISCUSSION THREAD

**Discussion by Rarder44** [alexandrainst/php-xlsx-fast-editor#25] [Q&A] [answered]
Hi, when I try to edit an empty file (newly created) the following error comes out:

Missing XML fragment xl/sharedStrings.xml

but if I open the file and write something on any page, the error is not shown.

**Comment by Alkarex** [comment, marked as answer]
Indeed, a blank Excel document seems to lack this structure.
For now, the workaround is to start with an Excel document in which you have at least written some text once (and it is fine to delete the text again)

**Comment by Rarder44** [comment]
Sorry for the question that might seem trivial, but can't it be created automatically in any way if it's missing?

**Comment by Alkarex** [comment]
It could be possible indeed, but several internal XML documents need to be updated, so not so straightforward and a bit risky. PR welcome, if anyone is motivated
